decidim-core/app/presenters/decidim/push_notification_presenter.rb

```diff
@@ -10,7 +10,8 @@ def title
     end
 
     def body
-      ActionView::Base.full_sanitizer.sanitize(event_class_instance.notification_title)
+      # Not using Rails sanitizers here because they escape HTML entities (i.e &amp;) and we want to keep them
+      event_class_instance.notification_title.gsub(%r{</?[^>]*>}, "") if event_class_instance.notification_title.present?
```

Comment:
This won't work with all text, e.g. take the following example:
```irb
> "There is a new comment from <a href='/path/to/profile'>Foo & Bar 1 < 2 & 2 > 3</a>".gsub(%r{</?[^>]*>}, "")
=> "There is a new comment from Foo & Bar 1  3"
```

But using Nokogiri might help here:
```irb
> Nokogiri::HTML("There is a new comment from <a href='/path/to/profile'>Foo & Bar 1 < 2 & 2 > 3</a>").text
=> "There is a new comment from Foo & Bar 1 < 2 & 2 > 3"
```

Not sure if we allow such characters in the notification body generally but I'd assume this as wide as possible.

It is also possible to improve the regular expression to cover all these cases but I'd trust these common parsers that are already available and we are also using elsewhere.

Loofah (which is essentially what the Rails sanitizer is using) would also work if you pass it the `encode_special_chars: false` flag but it can return weird outputs, e.g. adding extra line breaks to the places where it replaced block-level tags (such as `<p>`).